src/main/java/io/strimzi/kafka/quotas/StaticQuotaCallback.java

```diff
@@ -131,17 +158,39 @@ public void configure(Map<String, ?> configs) {
         storageQuotaHard = config.getHardStorageQuota();
         storageCheckInterval = config.getStorageCheckInterval();
         logDirs = Arrays.stream(config.getLogDirs().split(",")).map(Paths::get).collect(Collectors.toList());
-
-        log.info("Configured quota callback with {}. Storage quota (soft, hard): ({}, {}). Storage check interval: {}", quotaMap, storageQuotaSoft, storageQuotaHard, storageCheckInterval);
+        percentsMap = new HashMap<>();
+        String percents = config.getStoragePercents();
+        String[] split = percents.split("\\|");
+        for (String s : split) {
+            String[] inner = s.split(":");
+            if (inner.length != 3) {
+                throw new IllegalArgumentException("Invalid percent format: " + s);
+            }
+            Path path = Paths.get(inner[0]);
+            if (!logDirs.contains(path)) {
+                throw new IllegalArgumentException("Percents map contains path that's not part of logDirs: " + path + " - " + logDirs);
```

Comment:
The user doesn't know what "Percents map" is. You should refer to the config property name.